pkgs/by-name/du/dust/package.nix

```diff
@@ -1,22 +1,22 @@
 { stdenv, lib, fetchFromGitHub, rustPlatform, AppKit, installShellFiles }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "du-dust";
+  pname = "dust";
```

Comment:
Shouldn't `pname`'s be kept the same? Otherwise tools like `nix-env` that use this information could do unexpected things.

Reply:
Absolutely, you raise a valid point. If necessary, I'm prepared to revert this change.

Reply:
Yeah let's keep it the same then, ideally also add a small comment above the pname why it's different than the attribute name